parsl/executors/taskvine/exec_parsl_function.py

```diff
@@ -188,3 +173,19 @@ def execute_function(namespace, function_code, result_name):
         print("Could not write to result file.")
         traceback.print_exc()
         sys.exit(1)
+
+
+if __name__ == "__main__":
+    # parse the four required command line arguments:
+    # map_file: contains a pickled dictionary to map original names to
+    #           names at the execution site.
+    # function_file: contains the pickled parsl function to execute.
+    # argument_file: contains the pickled arguments to the function call.
+    # result_file: any output (including exceptions) will be written to
+    #              this file.
+    try:
+        (map_file, function_file, argument_file, result_file) = sys.argv[1:]
+    except ValueError:
+        print("Usage:\n\t{} function argument result mapping\n".format(sys.argv[0]))
+        raise
+    run(map_file, function_file, argument_file, result_file)
```

Comment:
You might consider wrapping this comment into a `usage()` function, so that this information can be shared with the hapless typo-ing user on the command line.  Consider:
```python
def usage(exc: Optional[Exception] = None):
    print("Usage: {} <mapping_path> <function_path> <args_path> <result_path>", file=sys.stderr)
    print("\n[Brief description of what this script does]", file=sys.stderr)
    print("  - mapping_path - path to a file containing a pickled dictionary ...", file=sys.stderr)
    print("  - function_path - ...", file=sys.stderr)
    
    if exc:
        print(f"\n({type(exc).__name}) {exc}", file=sys.stderr)

    sys.exit(1)  # non-zero exit code to represent "not successful" to the OS
```

Then can utilize the usage in place of the raise:
```python
try:
    map_file, function_file, argument_file, result_file = sys.argv[1:]
except ValueError as err:
    usage(err)  # won't return, per sys.exit
run(map_file, ...)
```